astroid/rebuilder.py

```diff
@@ -875,13 +875,11 @@ def visit_arguments(self, node: "ast.arguments", parent: NodeNG) -> nodes.Argume
             type_comment_posonlyargs=type_comment_posonlyargs,
         )
         # save argument names in locals:
+        if not newnode.parent:
+            raise ParentMissingError(target=newnode)
```

Comment:
```suggestion
        if not newnode.parent and (vararg or kwarg):
            raise ParentMissingError(target=newnode)
```
Otherwise I think this change the behavior ?